context/app/static/js/components/entity-search/searchkit-modifications/useSearchkitSDK.js

```diff
@@ -25,7 +25,7 @@ const useSearchkitSDK = ({ config, variables, filters, defaultSort }) => {
         const response = await request.execute({
           facets: true,
           hits: {
-            size: variables.page.size,
+            size: 18,
```

Comment:
This might be a good thing to pull from a higher-level configuration? But your call.